tests/test_flow/test_gather.py

```diff
@@ -213,6 +417,72 @@ def summary(df1, df2):
     }
 
 
+def test_multiple_gathers__unset_x(preset_builder):
+    builder = preset_builder
+
+    builder.clear_cases("x")
+
+    @builder
+    @bn.gather("w", "wx", "df1")
```

Comment:
I didn't realize we can use gather multiple times like this. This is pretty cool!